### ISSUE TITLE
BlackICE build fails

### ISSUE THREAD

**Issue by gururise** [revaldinho/opc#30]
try to build opc6 from the system/blackice/opc6 directory:
make -B build
```
---cut---
route...
  pass 1, 14 shared.
  pass 2, 3 shared.
  pass 3, 1 shared.
  pass 4, 0 shared.

After routing:
span_4     1891 / 29696
span_12    383 / 5632

  route time 4.90s
write_txt system.txt1...
icetime -d hx8k -P tq144:4k system.txt1
// Reading input .asc file..
// Reading 8k chipdb file..
// Creating timing netlist..
// Timing estimate: 24.38 ns (41.02 MHz)
icebram monitor_syn.mem monitor.mem < system.txt1 > system.txt2
Hexfiles have different number of words! (4096 vs. 0)
make: *** [../../../common/Makefile_ice40.inc:18: system.txt2] Error 1
```

**Comment by revaldinho** [comment]
Hi ,

Can you tell me what system you are running on please?

Rich

**Comment by gururise** [comment]
Nevermind.  I am on ArchLinux and it turns out I was missing xxd, causing monitor.txt not to be written.

**Comment by revaldinho** [comment]
ok. Good to hear you've got it working now.Rich                                                                                                                                                                        Sent from my BlackBerry                                                                                                                                                                            From: notifications@github.comSent: 25 September 2017 4:24 pmTo: opc@noreply.github.comReply to: reply@reply.github.comCc: richevans@mac.com; comment@noreply.github.comSubject: Re: [revaldinho/opc] BlackICE build fails (#30)  Closed #30.
